RedHat-UX/red-hat-design-system
# refactor: ScreenSizeController to use @rhds/tokens

## What I did

1. Refactored `ScreenSizeController` to use token values from `@rhds/tokens`
2. Updated `rh-navigation-secondary`, `rh-footer`, and `rh-stat` to use the new token keys.
3. Updated `rh-footer-responsive.css` made the values static.  

BLOCKED by https://github.com/patternfly/patternfly-elements/pull/2462

## Testing Instructions

1. View the [deploy preview](https://deploy-preview-875--red-hat-design-system.netlify.app/), make sure to check footer, secondary-nav and stat.
2. [navigation-secondary demo](https://deploy-preview-875--red-hat-design-system.netlify.app/elements/navigation-secondary/demo/)

## Notes to Reviewers

I think the `rh-footer-responsive.css` file could just go into a standard CSS file moving forward rather than being represented as a typescript file.  That said I added comments above each media query with the prior token and new token and if the values sync or not.  If there is a desire to add the interpolation back, feel free to make that commit.
